src/snapred/main.py

```diff
@@ -46,34 +40,37 @@ def __init__(self, parent):
         self.setMaximumHeight(ref_size + 2)
 
         btn_size = QSize(ref_size, ref_size)
-        for target in ('min', 'normal', 'max', 'close'):
+        for target in ("min", "normal", "max", "close"):
             btn = QToolButton(self, focusPolicy=Qt.NoFocus)
             layout.addWidget(btn)
             btn.setFixedSize(btn_size)
 
-            iconType = getattr(style, 
-                'SP_TitleBar{}Button'.format(target.capitalize()))
+            iconType = getattr(style, "SP_TitleBar{}Button".format(target.capitalize()))
             btn.setIcon(style.standardIcon(iconType))
 
-            if target == 'close':
-                colorNormal = 'red'
-                colorHover = 'orangered'
+            if target == "close":
+                colorNormal = "red"
+                colorHover = "orangered"
             else:
-                colorNormal = 'palette(mid)'
-                colorHover = 'palette(light)'
-            btn.setStyleSheet('''
+                colorNormal = "palette(mid)"
+                colorHover = "palette(light)"
+            btn.setStyleSheet(
+                """
                 QToolButton {{
                     background-color: {};
                 }}
                 QToolButton:hover {{
                     background-color: {}
                 }}
-            '''.format(colorNormal, colorHover))
+            """.format(
+                    colorNormal, colorHover
+                )
+            )
```

Comment:
I know this is trivial, but could we use an f-string?
```
            btn.setStyleSheet(
                f"""
                QToolButton {{
                    background-color: {colorNormal};
                }}
                QToolButton:hover {{
                    background-color: {colorHover)}
                }}
            '''
```
Also, why does one end in a `;` and the other doesn't?